babel.config.js

```diff
@@ -53,18 +53,6 @@ module.exports = {
     ['@babel/plugin-proposal-decorators', { legacy: true }],
     ['@babel/plugin-proposal-class-properties', { loose: true }],
     ['@babel/plugin-proposal-private-methods', { loose: true }],
-    [
-      '@babel/plugin-transform-runtime',
```

Comment:
Do we need this dependency and/or @babel/runtime-corejs3? If not, any others that should  be removed from dependencies as well?

Reply:
Good point, and I've now moved it down to the web side only (where it'll keep the bundle size small, in theory)